test/e2e/kibishii_tests.go

```diff
@@ -40,28 +39,28 @@ func installKibishii(ctx context.Context, namespace string, cloudPlatform string
 	// We use kustomize to generate YAML for Kibishii from the checked-in yaml directories
 	kibishiiInstallCmd := exec.CommandContext(ctx, "kubectl", "apply", "-n", namespace, "-k",
 		"github.com/vmware-tanzu-experiments/distributed-data-generator/kubernetes/yaml/"+cloudPlatform)
-
-	_, _, err := veleroexec.RunCommand(kibishiiInstallCmd)
+	stdout, stderr, err := veleroexec.RunCommand(kibishiiInstallCmd)
 	if err != nil {
+		fmt.Println(stdout)
+		fmt.Println(stderr)
```

Comment:
This is trivial but for consistency, how about handle the stdout/stderr following this convention?

https://github.com/reasonerjt/velero/blob/9ffffda11e90241b6b74e9d81b578e631b435a99/pkg/controller/pod_volume_restore_controller.go#L378